packages/audit/hooks/audit-route.py

```
@@ -178,7 +184,12 @@ def main() -> None:
 
     config = load_config(data)
     policy = load_package_config("audit", "delegation-policy.json", root)
-    all_aliases = {**build_aliases(config), **(policy.get("aliases") or {})}
+    # policy 側の aliases はキーごとに union する（丸ごと上書きすると
```

Ruff (0.15.20)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)